src/saltext/vmware/modules/vmc_org_users.py

```diff
@@ -0,0 +1,106 @@
+"""
+Salt execution module for VMC org users
+Provides methods to get, add and remove user within a org.
+"""
+import logging
+
+from saltext.vmware.utils import vmc_constants
+from saltext.vmware.utils import vmc_request
+from saltext.vmware.utils import vmc_templates
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = "vmc_org_users"
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def get(
```

Comment:
**suggestion** should be `list`

Reply:
Hi Wayne, we followed **get** for list and **get_by_id** for particular item in all the modules.
Do let me know if this need to be renamed , we will do througout .

Reply:
Yes - [that should be done throughout](https://github.com/saltstack/salt-ext-modules-vmware/issues/240)

Reply:
okay , i will make this change throughout

Reply:
@waynew just pushed the change